specutils/io/default_loaders/tabular_fits.py

```diff
@@ -181,8 +183,12 @@ def tabular_fits_writer(spectrum, file_name, hdu=1, update_header=False, **kwarg
         if columns[c].ndim > 1:
             columns[c] = columns[c].T
 
-    tab = Table(columns, names=colnames, meta=header)
+    tab = Table(columns, names=colnames)
+    hdu1 = fits.BinTableHDU(data=tab, header=header)
+    hdu1.header['EXTNAME'] = 'DATA'
+    hdu0 = fits.PrimaryHDU(header=header)
 
-    # Todo: support writing to other HDUs than the default (1st)
-    #       and an 'update' mode so different HDUs can be written to separately
-    tab.write(file_name, format="fits", **kwargs)
+    hdulist = fits.HDUList([hdu0, hdu1])
```

Comment:
Is writing to 2nd or subsequent extensions then no longer of interest?

Reply:
Writing more HDUs can be addressed in a future pull request. I'm happy to open an issue once this gets merged.